{"name": "test_check_order_number_in_the_work_panel_after_making_order[firefox]", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16", "trace": "self = <tests.test_order_feed_page.TestOrderFeedPage object at 0x102adcb10>\nbrowser = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"18207ce2-dfe9-41a0-8afa-aacd19d830ee\")>\n\n    def test_check_order_number_in_the_work_panel_after_making_order(self, browser):\n        main_page = MainPage(browser)\n        main_page.go_to_site(BasePageLocators.MAIN_PAGE_URL)\n        main_page.click_login_button()\n        login_page = LoginPage(browser)\n        login_page.input_email_field()\n        login_page.input_password_field()\n        login_page.click_enter_button()\n        main_page.drag_r2_d3_bun_to_constructor()\n        main_page.click_make_order_button()\n        main_page.check_order_status_text()\n        order_number_on_main = main_page.check_order_number_on_order_created_window()\n        main_page.click_close_window_detail_ingredient_button()\n        main_page.click_order_feed_button()\n        order_feed_page = OrderFeedPage(browser)\n>       order_number_on_feed = order_feed_page.check_order_number_in_the_work_panel()\n\ntests/test_order_feed_page.py:78: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/order_feed_page.py:44: in check_order_number_in_the_work_panel\n    header_text = self.find_button_located_hard(OrderFeedPageLocators.ORDER_NUMBER_IN_THE_WORK_PANEL).text\npages/base_page.py:35: in find_button_located_hard\n    element = WebDriverWait(self.browser, time).until(EC.visibility_of_element_located(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"18207ce2-dfe9-41a0-8afa-aacd19d830ee\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x102c31e40>, message = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nE       WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nE       NoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\nE       dom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n../../page_object_pythonProject/.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException"}, "parameters": [{"name": "browser", "value": "'firefox'"}], "start": 1723547714861, "stop": 1723547735618, "uuid": "e0632bc8-9c9e-48e2-b44d-af17ff80d5d7", "historyId": "107e6bc01279cf2957fa148dedc238f0", "testCaseId": "8cd1943bc98f84964817aef1ed87fcfe", "fullName": "tests.test_order_feed_page.TestOrderFeedPage#test_check_order_number_in_the_work_panel_after_making_order", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order_feed_page"}, {"name": "subSuite", "value": "TestOrderFeedPage"}, {"name": "host", "value": "MacBook-Pro-Ila.local"}, {"name": "thread", "value": "61939-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order_feed_page"}]}